test/fixtures/worker-preload.js

```diff
@@ -0,0 +1,6 @@
+const { Worker, isMainThread, workerData } = require('worker_threads');
+
+let count = isMainThread ? 0 : workerData + 1;
+if (workerData < 2) {
+  new Worker('1 + 1', { eval: true, count });
+}
```

Comment:
Did you mean to pass the count as `workerData` rather than an option on its own? (In this test, you’d have `count === threadId` anyway)

Reply:
Ha! yeah...